src/OrchardCore.Modules/OrchardCore.OpenId/Controllers/ApplicationController.cs

```diff
@@ -174,7 +176,8 @@ public async Task<IActionResult> Create(CreateOpenIdApplicationViewModel model,
                 RedirectUris = model.RedirectUris,
                 Roles = model.RoleEntries.Where(x => x.Selected).Select(x => x.Name).ToArray(),
                 Scopes = model.ScopeEntries.Where(x => x.Selected).Select(x => x.Name).ToArray(),
-                Type = model.Type
+                Type = model.Type,
+                RequirePkce = model.RequirePkce
```

Comment:
For consistency, we should probably name these properties `RequireProofKeyForCodeExchange`.

Reply:
Yup, make sense.  This has been done.